### PR TITLE
Added sql user creation using the new --host flag

### PR DESCRIPTION
**Pull request by venkayya**

### CODE REVIEW

File: scripts/postgres_instance.sh [hunk 1]
```diff
@@ -47,6 +47,6 @@ fi
 
 # Making a Postgres user that is allowed to connect from any host
 gcloud sql users create "$USER_NAME" \
-'%' \
+--host '%' \
```
Comment (chrislovecnm) [inline scripts/postgres_instance.sh]: Thanks

**Comment by chrislovecnm** [comment]
We need CI to run

**Comment by chrislovecnm** [comment]
please test this

**Comment by venkayya** [comment]
test this please

**Comment by venkayya** [comment]
test this

**Comment by emailbob** [comment]
test this please

**Pull request merged**